src/templates/token.tsx

```diff
@@ -132,5 +247,6 @@ export const generateTokenHtml = (
   { secret }: { secret: string },
   options = {
     validationLevel: 'soft',
+    minify: false,
```

Comment:
This caused spamming `minify is deprecated` in console every time somebody requests email